apps/server/src/modules/board/controller/api-test/content-element-create.api.spec.ts

```diff
@@ -142,6 +142,14 @@ describe(`content element create (api)`, () => {
 
 				expect(response.statusCode).toEqual(400);
 			});
+
+			it('should return the created content element of type DRAWING', async () => {
+				const { loggedInClient, cardNode } = await setup();
+
+				const response = await loggedInClient.post(`${cardNode.id}/elements`, { type: ContentElementType.DRAWING });
+
+				expect((response.body as AnyContentElementResponse).type).toEqual(ContentElementType.DRAWING);
+			});
```

Comment:
here you can also test that the actual entity was updarted, besides the response
look at line 313 below for an example

Reply:
added